### ISSUE TITLE
policies.json missing in directus-template-cli apply

### ISSUE THREAD

**Issue by vwasteels** [directus-labs/directus-template-cli#68]
after following setup tutorial on self hosted with docker compose :

```bash
vwasteels@indivisible : npx directus-template-cli@ apply  --content       
=== Directus Template CLI - Apply

? What type of template would you like to apply? Community templates
? Select a template. AgencyOS
You selected AgencyOS
------------------
What is your Directus URL? [http://localhost:8055]: 
? How do you want to log in? Directus Access Token
What is your Directus Admin Token?: XXXXXXXXXXXXXXXXXXXXXXX
-- Logged in as Admin User
=== Applying template - AgencyOS to http://localhost:8055

Loading 76 collections and 700 fields... done
Loading 176 relations... done
Loading 4 roles... done
 ›   Warning: Unknown error: "File not found: 
 ›   /Users/vwasteels/.npm/_npx/b3851e97124b8359/node_modules/directus-template-cli/dist/lib/downloads/official/agencyos/src/policies.json"
    Error: ENOENT: no such file or directory, open 
    '/Users/vwasteels/.npm/_npx/b3851e97124b8359/node_modules/directus-template-cli/dist/lib/downloads/official/agencyos/src/policies.json'
    Code: ENOENT
```

```
npm -v 
10.9.1

directus-template-cli@0.5.0
```

**Comment by den5o** [comment]
I tried this today as well and got the same error running it in a podman container.
`npm 10.8.2`
`directus-template-cli@0.5.0`

**Comment by bryantgillespie** [comment]
Hi all 👋
Sorry for any troubles here. The issue is not with the CLI but the current version of the AgencyOS template.

The v11 version of the AgencyOS template will be updated soon (days).

**Comment by bryantgillespie** [comment]
This should now be resolved with directus-template-cli v0.5.1

`npx directus-template-cli@latest apply`

There was also an issue where the CLI could potentially use a cached copy of the template - which could cause if you had tried to install a community template previously.
